src/__tests__/utils/password.test.ts

```diff
@@ -0,0 +1,25 @@
+import { checkHash, hashFunction } from '../../utils/password';
+import { getRandomString } from '../../__mocks__/getRandom';
+
+describe('testing the hashFunction: ', () => {
+  it('should compare the secret hashed and without', async () => {
+    const secret = getRandomString();
+    expect(await hashFunction(secret)).not.toBe(secret);
```

Comment:
this seems like it's testing just that the hash function works at all, which sounds like it should be a separate test